contracts/examples/NonMerklizedIdentityExample.sol

```diff
@@ -0,0 +1,152 @@
+   
+    // claimsMap claims storage
+    mapping(uint256 => mapping(string => ClaimInfo)) private claimsMap;
+   
+    uint256[44] private __gap_after;
+
+    function initialize(
+        address _stateContractAddr,
+        string calldata _jsonldSchemaURL, 
+        uint256 _jsonldSchemaHash,
+        string calldata _jsonSchemaURL,
+        string calldata _credentialType
+    ) public initializer {
+        jsonldSchemaURL = _jsonldSchemaURL;
+        jsonldSchemaHash = _jsonldSchemaHash;
+        jsonSchemaURL = _jsonSchemaURL;
+        credentialType = _credentialType;
+
+        IdentityBase.initialize(_stateContractAddr);
+        __Ownable_init();
+    }
+
+    // saveClaim save a claim to storage
+    function saveClaim(
+        uint256 _id,
+        string memory _uuid,
+        uint256[8] memory _claim
+    ) private {
+        claimsMap[_id][_uuid] = ClaimInfo({
+            jsonldSchemaURL: jsonldSchemaURL,
+            jsonldSchemaHash: jsonldSchemaHash,
+            jsonSchemaURL: jsonSchemaURL,
+            credentialType: credentialType,
+            claim: _claim
+        }); 
+        countOfIssuedClaims++;
+    }
+
+    /**
+     * @dev Get user claim by user id and uuid
+     * @param _userId - user id
+     * @param _uuid - uuid of the claim that was used for issuing claim
+     */
+    function getUserClaim(uint256 _userId, string memory _uuid) public view returns (ClaimInfo memory) {
+        return claimsMap[_userId][_uuid];
+    }
+
+    /**
+     * @dev Set schema
+     * @param _jsonldSchemaURL  - JSONLD schema url
+     * @param _jsonldSchemaHash - JSONLD schema hash. Use this code snippet to calculate hash: https://go.dev/play/p/3id7HAhf-Wi 
+     * @param _jsonSchemaURL  - JSON schema url
+     * @param _credentialType - credential type
+     */
+    function setSchema(
+            string memory _jsonldSchemaURL, 
+            uint256 _jsonldSchemaHash,
+            string memory _jsonSchemaURL,
+            string memory _credentialType
+        ) public onlyOwner {
+        jsonldSchemaURL = _jsonldSchemaURL;
+        jsonldSchemaHash = _jsonldSchemaHash;
+        jsonSchemaURL = _jsonSchemaURL;
+        credentialType = _credentialType;
+    }
+
+    // addClaimAndTransit add a claim to the identity and transit state
+    function addClaimAndTransit(uint256[8] memory _claim) private {
+        identity.addClaim(_claim);
+        identity.transitState();
+    }
+
+    /**
+     * @dev Revoke claim using it's revocationNonce
+     * @param _revocationNonce  - revocation nonce
+     */
+    function revokeClaimAndTransit(uint64 _revocationNonce) public onlyOwner {
+        identity.revokeClaim(_revocationNonce);
+        identity.transitState();
+    }
+
+    /**
+     * @dev Issue credential
+     * @param _userId - user id for which the claim is issued
+     * @param _uuid - uuid of the claim
+     */
+    function issueCredential(uint256 _userId, string memory _uuid) public {
```

Comment:
calldata here for _uuid

Reply:
I use _uuid to get a specific core claim that was issued for the user.